tests/integration/grpc_test.go

```diff
@@ -154,7 +154,7 @@ func templateEndpoints(t *testing.T, pattern string, clus *integration.Cluster)
 	for _, m := range clus.Members {
 		ent := pattern
 		if strings.Contains(ent, "%d") {
-			ent = fmt.Sprintf(ent, integration.GrpcPortNumber(m.UniqNumber, m.MemberNumber))
+			ent = fmt.Sprintf(strings.Replace(ent, "%d", "%s", -1), m.GrpcPortNumber())
```

Comment:
That turned out a bit weird due to the change in type and since we're checking for %s below as well, this has some unexpected clashes.

Reply:
Why not update patterns to change `%d` to `%s`

Reply:
unfortunately this test controls the input by which type of pattern is used:
https://github.com/etcd-io/etcd/blob/ba9f5b557f1dfca3444259c587fb751ee238b031/tests/integration/grpc_test.go#L156-L161

so when I would change it it would always fill it with m.Name.
Is it worth to refactor the whole test to make this work, wdyt?

Reply:
ah, ok it's terrible :(

Reply:
```suggestion
			ent = fmt.Sprintf(ent, strconv.Atoi(m.GrpcPortNumber()))
```
What about this?

Reply:
great idea, implemented.